app/build.gradle

```diff
@@ -6,8 +6,8 @@ def configuration = rootProject.ext.configuration
 def libraries = rootProject.ext.libraries
 
 android {
-    compileSdkVersion configuration.compileSdkVersion
-    buildToolsVersion configuration.buildToolsVersion
+    compileSdkVersion 28
```

Comment:
better if you put all the dependencies' version into one file(build.gradle of project is a good option) 😉

Reply:
Just changed 😄